<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>965</width>
    <height>559</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ПО для настройки БГС</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="btnConnect">
      <property name="text">
       <string>Получить список устройств</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="gbDevices">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>150</height>
       </size>
      </property>
      <property name="title">
       <string>Устройства</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>1</number>
       </property>
       <property name="topMargin">
        <number>1</number>
       </property>
       <property name="rightMargin">
        <number>1</number>
       </property>
       <property name="bottomMargin">
        <number>1</number>
       </property>
       <item>
        <widget class="QListView" name="lvDevices">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnCreate">
         <property name="text">
          <string>Установить связь</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frMain">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QGroupBox" name="gbData">
         <property name="title">
          <string>Данные</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="leftMargin">
           <number>1</number>
          </property>
          <property name="topMargin">
           <number>1</number>
          </property>
          <property name="rightMargin">
           <number>1</number>
          </property>
          <property name="bottomMargin">
           <number>1</number>
          </property>
          <item>
           <widget class="QTextEdit" name="teData">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="gbCommand">
         <property name="title">
          <string>Команды</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="lblName">
            <property name="text">
             <string>Номер устройства</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="edName">
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="maxLength">
             <number>4</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnSetName">
            <property name="text">
             <string>Записать</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btnSetBaud4">
            <property name="text">
             <string>BAUD 4</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnSetPio11">
            <property name="text">
             <string>PIO 11</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="gbControl">
         <property name="title">
          <string>Управление</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QFrame" name="frPower">
            <property name="frameShape">
             <enum>QFrame::Panel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Sunken</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <property name="spacing">
              <number>1</number>
             </property>
             <property name="leftMargin">
              <number>1</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>1</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="lblPower">
               <property name="text">
                <string>Мощность</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="frPowerButtons">
               <property name="frameShape">
                <enum>QFrame::NoFrame</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <property name="leftMargin">
                 <number>1</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>1</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QPushButton" name="btnUpValue">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>50</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>+</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="btnDownValue">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>50</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>-</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frModulation">
            <property name="frameShape">
             <enum>QFrame::Panel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Sunken</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <property name="spacing">
              <number>2</number>
             </property>
             <property name="leftMargin">
              <number>1</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>1</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="lblModulation">
               <property name="text">
                <string>Модуляция</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="frModulationCtrl">
               <property name="frameShape">
                <enum>QFrame::NoFrame</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QComboBox" name="cbModulation">
                  <property name="minimumSize">
                   <size>
                    <width>120</width>
                    <height>0</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="btnModulationSet">
                  <property name="text">
                   <string>Применить</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frFrequency">
            <property name="frameShape">
             <enum>QFrame::Panel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Sunken</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <property name="spacing">
              <number>2</number>
             </property>
             <property name="leftMargin">
              <number>1</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>1</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="lblFrequency">
               <property name="text">
                <string>Частота</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="frFrequencyCtrl">
               <property name="frameShape">
                <enum>QFrame::NoFrame</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QComboBox" name="cbFrequency">
                  <property name="minimumSize">
                   <size>
                    <width>120</width>
                    <height>0</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="btnFrequencySet">
                  <property name="text">
                   <string>Применить</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frIntensivity">
            <property name="frameShape">
             <enum>QFrame::Panel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Sunken</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_8">
             <property name="spacing">
              <number>2</number>
             </property>
             <property name="leftMargin">
              <number>1</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>1</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="lblIntensivity">
               <property name="text">
                <string>Интенсивность</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="frIntensivityCtrl">
               <property name="frameShape">
                <enum>QFrame::NoFrame</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_6">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QComboBox" name="cbIntensivity">
                  <property name="minimumSize">
                   <size>
                    <width>120</width>
                    <height>0</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="btnIntensivitySet">
                  <property name="text">
                   <string>Применить</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frModulation_2">
            <property name="frameShape">
             <enum>QFrame::Panel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Sunken</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_9">
             <property name="spacing">
              <number>2</number>
             </property>
             <property name="leftMargin">
              <number>1</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>1</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="lblModulation_2">
               <property name="text">
                <string>Команда, hex</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="frModulationCtrl_2">
               <property name="frameShape">
                <enum>QFrame::NoFrame</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_7">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QLineEdit" name="edCmd">
                  <property name="font">
                   <font>
                    <pointsize>10</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="btnModulationSet_2">
                  <property name="text">
                   <string>Применить</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>965</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>btnConnect</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_connect()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>155</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>519</x>
     <y>55</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lvDevices</sender>
   <signal>activated(QModelIndex)</signal>
   <receiver>MainWindow</receiver>
   <slot>on_selectDevice(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>157</x>
     <y>170</y>
    </hint>
    <hint type="destinationlabel">
     <x>541</x>
     <y>87</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lvDevices</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>MainWindow</receiver>
   <slot>on_selectDevice(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>165</x>
     <y>154</y>
    </hint>
    <hint type="destinationlabel">
     <x>543</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnUpValue</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_valueUp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>83</x>
     <y>521</y>
    </hint>
    <hint type="destinationlabel">
     <x>628</x>
     <y>268</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnDownValue</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_valueDown()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>139</x>
     <y>521</y>
    </hint>
    <hint type="destinationlabel">
     <x>639</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCreate</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_connectDevice()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>209</x>
     <y>261</y>
    </hint>
    <hint type="destinationlabel">
     <x>633</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnSetName</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_setName()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>256</x>
     <y>418</y>
    </hint>
    <hint type="destinationlabel">
     <x>847</x>
     <y>286</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnSetBaud4</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_setBaud()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>375</x>
     <y>419</y>
    </hint>
    <hint type="destinationlabel">
     <x>839</x>
     <y>311</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnSetPio11</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_setPIO()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>472</x>
     <y>424</y>
    </hint>
    <hint type="destinationlabel">
     <x>839</x>
     <y>337</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnModulationSet</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_setModulation()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>317</x>
     <y>506</y>
    </hint>
    <hint type="destinationlabel">
     <x>843</x>
     <y>363</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnFrequencySet</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_setFrequency()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>504</x>
     <y>513</y>
    </hint>
    <hint type="destinationlabel">
     <x>843</x>
     <y>389</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnIntensivitySet</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_setIntensivity()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>745</x>
     <y>508</y>
    </hint>
    <hint type="destinationlabel">
     <x>843</x>
     <y>416</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnModulationSet_2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_sendAnyCommand()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>866</x>
     <y>507</y>
    </hint>
    <hint type="destinationlabel">
     <x>950</x>
     <y>411</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_connect()</slot>
  <slot>on_selectDevice(QModelIndex)</slot>
  <slot>on_create()</slot>
  <slot>on_command()</slot>
  <slot>on_valueUp()</slot>
  <slot>on_valueDown()</slot>
  <slot>on_connectDevice()</slot>
  <slot>on_setName()</slot>
  <slot>on_setBaud()</slot>
  <slot>on_setPIO()</slot>
  <slot>on_setModulation()</slot>
  <slot>on_setFrequency()</slot>
  <slot>on_setIntensivity()</slot>
  <slot>on_sendAnyCommand()</slot>
 </slots>
</ui>
